### PR TITLE
Bug 57 - Make priority sidebar collapsable

### PR DESCRIPTION
**Pull request by AllegroFox**
It toggles! Still have to work on replacing the button with a nice icon.

### CODE REVIEW

File: src/content/components/PriorityGuide/PriorityGuide.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
 .container {
   padding: 20px;
-  background: #f9f9fa;
   width: 250px;
```
Comment (k88hudson) [inline src/content/components/PriorityGuide/PriorityGuide.scss]: You're also going to want to transition the width so this shrinks along with the inner content

File: src/content/components/PriorityGuide/PriorityGuide.scss [hunk 2]
```diff
@@ -34,3 +33,57 @@
...
+.active {
+  background: #f9f9fa;
+  padding: 20px;
+  position: absolute;
+  width: 250px;
+  top: 0px;
+  right: 0;
+  bottom: 0;
+
+  -moz-transition: right 0.25s ease;
```
Comment (k88hudson) [inline src/content/components/PriorityGuide/PriorityGuide.scss]: no need for `moz-transition`! I hope no one is using <Firefox 16

Comment (k88hudson) [reply src/content/components/PriorityGuide/PriorityGuide.scss]: Also,  using `translateX` here is better for performance than using `right` (see this article for more if you're interested https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/)

Comment (AllegroFox) [reply src/content/components/PriorityGuide/PriorityGuide.scss]: Is that something you'd like me to change? Or just an FYI :)

Comment (k88hudson) [reply src/content/components/PriorityGuide/PriorityGuide.scss]: Please change yeah

Comment (AllegroFox) [reply src/content/components/PriorityGuide/PriorityGuide.scss]: okay, will do!

**Pull request merged**